api/utils.py

```diff
@@ -92,7 +92,10 @@ def get_col(sqlalch_obj, serializer=False):
             field_cls = col_prop.columns[0].type.__class__
             if not field_nm.startswith('_'):
                 if serializer:
-                    d.append((field_nm, field_mapping[field_cls.__visit_name__]()))
+                    if 'datetime' in field_nm.lower() and 'utcoffset' not in field_nm.lower():
+                        d.append((field_nm, DateTimeField()))
+                    else:
+                        d.append((field_nm, field_mapping[field_cls.__visit_name__]()))
```

Comment:
I think this test will handle assigning `*utcoffset` fields to `IntegerField`, right? (from L71)

Also, if I'm right in my other, general comments, DateTime fields are only assigned to variant type for SQLite databases. So, all this machinery (L94-L97) is probably unnecessary for the other 3 database types ... which are the ones most likely to serve as backends for the REST API.